app/lib/widgets/location/location_screen.dart

```diff
@@ -210,31 +308,37 @@ class _LocationScreenState extends State<LocationScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
         appBar: AppBar(
-            backgroundColor: Theme.of(context).colorScheme.inversePrimary,
-            title: Text(AppLocalizations.of(context)!.locationScreenName),
-            actions: [
-              if (kIsWeb)
-                IconButton(
-                    onPressed: _refreshData,
-                    icon: const Icon(ManvIcons.refresh))
-            ]),
+          backgroundColor: Theme.of(context).colorScheme.inversePrimary,
+          title: Text(AppLocalizations.of(context)!.locationScreenName),
+          actions: [
+            if (kIsWeb)
+              IconButton(
+                  onPressed: _refreshData, icon: const Icon(ManvIcons.refresh))
+          ],
+          bottom: TabBar(
+            controller: _tabController,
+            tabs: [
+              _buildTab(AppLocalizations.of(context)!.locationScreenOverview,
+                  ManvIcons.patient),
+              _buildTab(AppLocalizations.of(context)!.locationScreenResources,
+                  Icons.shopping_bag),
+            ],
+          ),
+        ),
         body: CustomFutureBuilder<Location>(
             future: _futureLocation,
             builder: (context, location) {
               _fetchedLocation = location;
 
-              return RefreshIndicator(
-                  onRefresh: _refreshData,
-                  child: SingleChildScrollView(
-                      physics: const AlwaysScrollableScrollPhysics(),
-                      child: Column(children: [
-                        Card(child: LocationOverview(location: location)),
-                        Text(AppLocalizations.of(context)!
-                            .locationScreenAvailableSubLocations),
-                        _buildLocationDirectory(location),
-                        if (_showInventory) _buildInventory(),
-                      ])));
+              return TabBarView(
+                controller: _tabController,
+                children: [
+                  _buildOverview(location),
+                  _buildTransfer(location),
+                ],
+              );
             }),
-        bottomNavigationBar: _buildButtonBar());
+        bottomNavigationBar:
+            _tabController.index == 1 ? _buildButtonBar() : null);
```

Comment:
Couldn't you use DefaultTabController because of this?  
Alternatively we could also just add it to the bottom of the transfer tab

Reply:
I think its more complicated to add it to the bottom than using this controller as the widget is scrollable